services/postfix/mailman.nix

```diff
@@ -0,0 +1,158 @@
+# Manual steps post-deploy:
```

Comment:
i assume one of if the case ensure it's in the docs

Reply:
Will be doc'd, along with the rest of the setup